index.html

```diff
@@ -3281,6 +3281,13 @@ <h3>Reserved Extension Points</h3>
 reserved property. For more information related to adding `type` information,
 see Section <a href="#types"></a>.
         </p>
+        <p class="issue" data-number="882">
```

Comment:
```suggestion
        <p class="issue" title="Extension points under consideration by the Working Group">
```

Reply:
I'd be a +1 to this PR if the change above is made.

Reply:
is there a way to have both title and data number?